server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -335,6 +337,39 @@ function getContainerName($siteId) {
 	return 'server-contingency-site';
 }
 
+function getToEmailString($siteId) {
```

Comment:
Not crucial for this PR, but maybe it's worth adding an `email` column on the `Site` table? Although, maybe it's something that we don't want site coordinators to be able to change easily just in case they mess up, and it goes to a defunct/incorrect email